app/src/main/java/cm/aptoide/pt/install/AutoUpdate.java

```diff
@@ -139,7 +139,7 @@ private void requestUpdateSelf(final AutoUpdateInfo autoUpdateInfo) {
     AptoideApplication.setAutoUpdateWasCalled(true);
 
     ContextThemeWrapper wrapper = new ContextThemeWrapper(activity,
-        activity.obtainStyledAttributes(new int[] { R.attr.alertDialog })
+        activity.obtainStyledAttributes(new int[] { R.attr.dialogTheme })
```

Comment:
After the this, the alertDialog attribute is not used anymore so it should be removed